File: wrappers/mvc/src/Kendo.Mvc/UI/Sortable/SortableCursorOffset.cs
namespace Kendo.Mvc.UI
{
    public class SortableCursorOffset
    {
        public SortableCursorOffset()
        {
            Top = int.MinValue;
            Left = int.MinValue;
        }

        public int Top { get; set; }
        public int Left { get; set; }
    }
}

File: wrappers/php/lib/Kendo/UI/SortableCursorOffset.php
<?php

namespace Kendo\UI;

class SortableCursorOffset extends \Kendo\SerializableObject {
//>> Properties

    /**
    * The left offset of the hint element relative to the mouse cursor/finger.
    * @param float $value
    * @return \Kendo\UI\SortableCursorOffset
    */
    public function left($value) {
        return $this->setProperty('left', $value);
    }

    /**
    * The top offset of the hint element relative to the mouse cursor/finger.
    * @param float $value
    * @return \Kendo\UI\SortableCursorOffset
    */
    public function top($value) {
        return $this->setProperty('top', $value);
    }

//<< Properties
}

?>
